ci/build_wheel.sh

```diff
@@ -45,6 +45,10 @@ cd "${package_dir}"
 
 python -m pip wheel . -w dist -vvv --no-deps --disable-pip-version-check
 
+# Fix Python wheel tag to be restricted to CPython and version dependent
+WHEEL_PYTHON_TAG=`python -c 'import sys; print("cp{0}{1}".format(*sys.version_info[:2]))'`
```

Comment:
This is an alternate way to achieve the same thing, but I am fine with it as-is
```suggestion
WHEEL_PYTHON_TAG=cp$(echo ${RAPIDS_PY_VERSION} | sed 's/\.//g')
```

Reply:
No strong feelings between the other approaches. Will leave for now and see what others think

Reply:
I'm in favor of using `RAPIDS_PY_VERSION` (accepting this suggestion).